readme.md

```diff
@@ -38,10 +38,30 @@ The source object to copy properties from.
 
 #### mapper
 
-Type: `(sourceKey, sourceValue, source) => [targetKey, targetValue, mapperOptions?] | mapObjectSkip`
+Type: `(sourceKey, sourceValue, source, path) => [targetKey, targetValue, mapperOptions?] | mapObjectSkip`
 
 A mapping function.
 
+##### path
```

Comment:
The docs here are missing in index.d.ts. Readme and index.d.ts should be in sync docs-wise.